route53_readysetcyber_app.tf

```
@@ -46,6 +46,18 @@ resource "aws_route53_record" "ready_set_cyber_prod_AAAA" {
   zone_id = aws_route53_zone.cyber_dhs_gov.zone_id
 }
 
+resource "aws_route53_record" "readysetcyber_cd_acme_TXT" {
```

Be consistent with other RSC resource names:
```suggestion
resource "aws_route53_record" "ready_set_cyber_cd_acme_TXT" {
```